--- php-pecl-ares-0.7.0/ares-0.7.0/ares.c~	1970-01-01 10:13:08.000000000 +0100
+++ php-pecl-ares-0.7.0/ares-0.7.0/ares.c	2010-01-03 23:14:19.610056783 +0100
@@ -758,7 +758,7 @@
 		
 		MAKE_STD_ZVAL(parsed);
 		ZVAL_NULL(parsed);
-		if (SUCCESS == php_ares_parse(abuf, alen, parsed)) {
+		if (SUCCESS == php_ares_parse(abuf, alen, parsed TSRMLS_CC)) {
 			q->result.std.arr = parsed;
 		} else {
 			zval_ptr_dtor(&parsed);
